src/microfrontends/assetManifestUtils.test.ts

```diff
@@ -1,29 +1,24 @@
 import { makeAbsolute } from './assetManifestUtils';
 import { SAMTALESTØTTE_MIKROFRONTEND_PATH } from '../konstanter';
-import { withCurrentLocation } from "../utils/test-utils";
 
 describe('makeAbsolute oppdaterer path til ressurser fra asset-manifest avhengig av domene child app og parent app kjører i', () => {
     it('returnerer samme path når begge kjører på samme domene', () => {
-        withCurrentLocation('http://dummy.io/containerapp', () => {
-            expect(
-                makeAbsolute(
-                    '/samtalestotte-podlet',
-                    '/samtalestotte-podlet/static/css/main.a617e044.chunk.css'
-                )
-            ).toBe('http://dummy.io/samtalestotte-podlet/static/css/main.a617e044.chunk.css');
-        });
+        expect(
```

Comment:
Var det noe grunn for å fjerne withCurrentLocation(...)

Reply:
Testen feilet når vi brukte withCurrentLocation, endte opp med å være localhost uansett.

Reply:
🤔

Reply:
feilet testene lokalt eller på GithubActions? 
Det funker lokalt hos meg foreløpig(med withCurrentLocation).?